### PR TITLE
Added new test method

### PR DESCRIPTION
**Pull request by tilakbbasnet**

### CODE REVIEW

File: src/app/pages/home/home.component.ts [hunk 1]
```diff
@@ -7,8 +7,18 @@ import { Component } from '@angular/core';
...
-export class HomeComponent {
-  execute(): void {
+export class HomeComponent implements OnInit{
+  execute(name: string): void {
+    console.log('Make appointment branch is executing '+name);
     console.log('Master branch is executing');
+    
+  }
+
+  test(): void {
```
Comment (poudelg25) [inline src/app/pages/home/home.component.ts]: Please rename this method.

File: src/app/pages/home/home.component.ts [hunk 2]
```diff
@@ -7,8 +7,18 @@ import { Component } from '@angular/core';
   templateUrl: './home.component.html',
   styleUrl: './home.component.scss'
 })
-export class HomeComponent {
-  execute(): void {
+export class HomeComponent implements OnInit{
+  execute(name: string): void {
+    console.log('Make appointment branch is executing '+name);
```
Comment (poudelg25) [inline src/app/pages/home/home.component.ts]: Remove console.log statement

File: src/app/pages/home/home.component.ts (hunk 1 above)
Comment (tilakbbasnet) [reply src/app/pages/home/home.component.ts]: Sure I will rename this method.

File: src/app/pages/home/home.component.ts (hunk 2 above)
Comment (tilakbbasnet) [reply src/app/pages/home/home.component.ts]: Sure!

File: src/app/pages/home/home.component.ts (hunk 1 above)
Comment (tilakbbasnet) [reply src/app/pages/home/home.component.ts]: Done, please review.

File: src/app/pages/home/home.component.ts (hunk 2 above)
Comment (tilakbbasnet) [reply src/app/pages/home/home.component.ts]: Done, please review.

**Review by poudelg25 — APPROVED** [review]
It looks good now.

**Pull request merged**